src/Symfony/Component/Validator/Mapping/Loader/PropertyInfoLoader.php

```diff
@@ -53,6 +56,10 @@ public function loadClassMetadata(ClassMetadata $metadata)
         }
 
         foreach ($properties as $property) {
+            if (false === $this->accessExtractor->isWritable($className, $property)) {
```

Comment:
`if (!...)`?

Reply:
No because the return value can be null (not guessed).